src/fobnail-attester.c

```diff
@@ -45,8 +74,108 @@ typedef struct {
 #endif
     attestation_context_t ac;
     UsefulBuf buf;
+    cmd_t cmds[MAX_CMDS + 1]; // last cmd must be NO_CMD
 } app_data_t;
 
+/* Some ANSI escape codes to make it more readable for users. */
+#define _N "\x1b[0m"
+#define _B "\x1b[1m"
+#define _U "\x1b[4m"
+
+static void usage(char *progname)
+{
+    fprintf(stderr, "\
+Usage:\n\
+    "_B"%s"_N" ["_B"cmd"_N" ["_B"cmd"_N"]]\n\
```

Comment:
Shouldn't this be
```
Usage:
    bin/fobnail-attester [CMD]...
```
Instead of current
```
Usage:
    bin/fobnail-attester [cmd [cmd]]
```
Proposed syntax would be closer to what `ls` and other tools (`grep`, `xargs`) do